test/modules/linear_boltzmann_solvers/transport_steady/reed_balance.lua

```diff
@@ -0,0 +1,100 @@
+-- Standard Reed 1D 1-group problem
+
+-- Create Mesh
+widths = {2., 1., 2., 1., 2.}
+nrefs  = {200,200,200,200,200}
+
+Nmat = #widths
+
+nodes = {}
+counter = 1
+nodes[counter] = 0.
+for imat = 1, Nmat do
+    dx = widths[imat]/nrefs[imat]
+    for i = 1, nrefs[imat] do
+        counter = counter + 1
+        nodes[counter] = nodes[counter-1] + dx
+    end
+end
+
+meshgen = mesh.OrthogonalMeshGenerator.Create({ node_sets = {nodes} })
+mesh.MeshGenerator.Execute(meshgen)
+
+-- Set Material IDs
+z_min = 0.0
+z_max = widths[1]
+for imat = 1, Nmat do
+    z_max = z_min + widths[imat]
+    print('imat, zmin, zmax', imat,z_min,z_max)
```

Comment:
We should use `log.Log(LOG_0, <string>)` here.

Reply:
fixed. Look at it and let me know if this is better. Is that option better just because of the potential for repeated console printouts in parallel? Just wondering. If so, then by all means we should use log.Log().

Reply:
I'm not 100% sure what the behavior of `print` in parallel is, but `log.Log` prints output in the same way as the codebase and has more optionality. For example, `print("Hello World!")` will print a line that looks like (maybe multiple times in parallel):
```
Hello World!
```
On the other hand `log.Log(LOG_0, "Hello World!")` will print a line that looks like:
```
[0]  Hello World!
```
which is in line with the rest of the OpenSn output.